tests/test_flake8_spellcheck.py

```diff
@@ -43,307 +43,309 @@ def test_is_number(value, result):
+        (flake8_path / "example.py").write_text("foo = 'bar'  # noqa: W503\n")
+        result = flake8_path.run_flake8()
         assert result.out_lines == []
 
-    def test_flake8_pragma_spaces(self, flake8dir):
-        flake8dir.make_example_py("foo = 'bar'  #    noqa: W503")
-        result = flake8dir.run_flake8()
+    def test_flake8_pragma_spaces(self, flake8_path):
+        (flake8_path / "example.py").write_text("foo = 'bar'  #    noqa: W503\n")
+        result = flake8_path.run_flake8()
         assert result.out_lines == [
             "./example.py:1:14: E262 inline comment should start with '# '"
         ]
 
     # Regression test for github.com/MichaelAquilina/flake8-spellcheck/issues/34
-    def test_empty_comment(self, flake8dir):
-        flake8dir.make_example_py(
+    def test_empty_comment(self, flake8_path):
+        (flake8_path / "example.py").write_text(
             """
-            # the empty comment below should not fail
-            #
-            # hello world
-        """
+# the empty comment below should not fail
+#
+# hello world
+"""
         )
-        result = flake8dir.run_flake8()
+        result = flake8_path.run_flake8()
         assert result.out_lines == []
 
     # Regression test for https://github.com/MichaelAquilina/flake8-spellcheck/issues/36
-    def test_type_and_noqa(self, flake8dir):
-        flake8dir.make_example_py(
+    def test_type_and_noqa(self, flake8_path):
+        (flake8_path / "example.py").write_text(
             """
-            # the comment below should not fail on `W503` or `W504`  # noqa: SC100
-            foo = []  # type: ignore  # noqa: W503  # noqa: W504
-        """
+# the comment below should not fail on `W503` or `W504`  # noqa: SC100
+foo = []  # type: ignore  # noqa: W503  # noqa: W504
+"""
         )
-        result = flake8dir.run_flake8()
+        result = flake8_path.run_flake8()
         assert result.out_lines == []
 
     # Regression test for github.com/MichaelAquilina/flake8-spellcheck/issues/40
-    def test_pure_number_char_comment(self, flake8dir):
-        flake8dir.make_example_py(
+    def test_pure_number_char_comment(self, flake8_path):
+        (flake8_path / "example.py").write_text(
             """
-            # the comment below should not fail
-            ##########
-            # hello world
-        """
+# the comment below should not fail
+##########
+# hello world
+"""
         )
-        result = flake8dir.run_flake8()
+        result = flake8_path.run_flake8()
         assert result.out_lines == []
 
 
 class TestFunctionDef:
-    def test_apostrophe(self, flake8dir):
-        flake8dir.make_example_py(
+    def test_apostrophe(self, flake8_path):
+        (flake8_path / "example.py").write_text(
             """
-            def dont_fail(a):
-                return a + 2
+def dont_fail(a):
+    return a + 2
 
 
-            def cant_fail(b):
-                return b * 4
-        """
+def cant_fail(b):
+    return b * 4
+"""
```

Comment:
Is this something that changed from flake8-dir and flake8-path?

I understand why it might be required, but it does make the tests a lot less nice to read :(

It's something we can solve at a later time regardless

Reply:
Without the left shift they all fail with indention errors. The older package must have manipulated the text.